briefcase/django.py

```diff
@@ -78,10 +82,48 @@ def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
         print("   - Installing NPM requirements...")
-
-        npm = shutil.which("npm")
-        subprocess.Popen([npm, "install"], cwd=os.path.abspath(self.dir)).wait()
-
+        node_attributes = {
+            'darwin': {
+                'url':'https://nodejs.org/dist/v6.10.3/node-v6.10.3.pkg',
+                'command_prefix': 'open '
+            },
+            'linux' : '',
+            'win32' : {
+                'url' :'https://nodejs.org/dist/v6.10.3/node-v6.10.3-x86.msi',
+                'command_prefix': 'msiexec /i '
+            }
+        }
+        system_platform = sys.platform
+        node = shutil.which('node')
+        npm  = shutil.which("npm")
+        try:
+            node_version_unicode = subprocess.check_output([node, '--version'])
+            node_version = node_version_unicode.decode('utf-8')
+            print('Node version %s detected' % node_version)
+        except:
+            node_version = ''
+            print('No version of Node detected\n Installing NodeJS... \nPlease Follow the instructions.')
+        if node and node_version[1] == '6':
+            subprocess.Popen(['npm', 'install'], cwd=os.path.abspath(self.dir)).wait()
+        elif node and node_version[1] != '6':
+            err_message = ('ERROR: Cannot run with the current version of Node and Npm, please \n'
+                           'unnistall the current version and install Node version 6.x\n'
+                           'Installation cancelled.'
+            )
+            raise RuntimeError(err_message)
+        else:
+            ses = requests.Session()
```

Comment:
I can take out the installation process, but, should I raise an error of no version of node is found?? or just an warning, print a link that may correspond to their platform and continue with the process??